src/main/java/com/wfector/command/CommandRunner.java

```diff
@@ -17,6 +16,7 @@
 
 public class CommandRunner implements CommandExecutor {
     private final ChestShopNotifier plugin;
+    private Help Help;
```

Comment:
Imo. there is no need to store the Help command in a separate field. IT can be instantiated when the command is used.

Reply:
How could i achieve that? if i remove that line the class would break when trying to get something from Help.

Reply:
You don't need the field (that you don't assign anywhere anyways) if you create a new object instance when it is needed. That would be more obvious if you used the normal, lower-case starting naming scheme of variables, fields and methods.

Reply:
Im not exactly sure what i must do now, do i need to remove the uppercase Help to help? And change that on line 114 too?

Reply:
Yes, instantiate a new Help object on line 114 too and remove the field, it never gets used nor assigned anywhere.